crates/workspace/src/persistence.rs

```diff
@@ -709,6 +863,37 @@ impl WorkspaceDb {
         }
     }
 
+    // TODO: Fix this query
```

Comment:
This is what you want to fix in a new PR right? So you can open all the breakpoints in a multi buffer?